http/api_handler.go

```diff
@@ -364,7 +365,7 @@ func (h *APIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if strings.HasPrefix(r.URL.Path, "/api/v2/telegrafs") {
+	if strings.HasPrefix(r.URL.Path, "/api/v2/telegraf") {
```

Comment:
thinking this needs to have an addiitinal check here since we'll have both routes available now and the order matters

```go
if strings.HasPrefix(r.URL.Path, "/api/v2/telegrafs") {
... stuffs
}

if strings.HasPrefix(r.URL.Path, "/api/v2/telegraf") {
```

Reply:
Agreed, i think the plugins logic should use it's own handler too.